core/modules/utils/filesystem.js

```diff
@@ -482,8 +482,10 @@ exports.deleteTiddlerFile = function(fileInfo,callback) {
 	//Only attempt to delete files that exist on disk
 	if(!fileInfo.filepath || !fs.existsSync(fileInfo.filepath)) {
 		//For some reason, the tiddler is only in memory or we can't modify the file at this path
-		$tw.syncer.displayError("Server deleteTiddlerFile task failed for filepath: "+fileInfo.filepath);
-		return callback(null,fileInfo);
+		let err = new Error("Non-existant filepath: "+fileInfo.filepath)
```

Comment:
Does the wording "Non-existant filepath" capture all the possible reasons for an error here? For example, it might be a misleading message for access control errors.